### ISSUE TITLE
empty json body ignored

### ISSUE THREAD

**Issue by davidhyman** [getsentry/responses#146]
Setting an empty json object (which is valid json) is not parsed as json by responses:
```
responses.add(
            responses.POST,
            url,
            json={},
            status=201
        )
```
In `responses.RequestsMock.add`:
```
        # if we were passed a `json` argument,
        # override the body and content_type
        if json:
            body = json_module.dumps(json)
            content_type = 'application/json'
```
Should probably be:
```
        # if we were passed a `json` argument,
        # override the body and content_type
        if json is not None:
            body = json_module.dumps(json)
            content_type = 'application/json'
```

**Comment by olipratt** [comment]
Hi,

I've been making good use of `responses` lately, and have also just been bitten by this bug. I can see the suggestion made above by @davidhyman was actually fixed in 9a3fbe65fbb565eabfb5cfc7afad57ed8edfc0cd back in Jan 2016, and the [latest release on PyPI](https://pypi.python.org/pypi/responses/0.5.1#downloads) is from a few days before that - can we get an updated release on PyPI with this fix in?

Thanks!

**Comment by dcramer** [comment]
This is resolved in master and we'll do a release shortly
